WorkingWithText/Program.cs

```diff
@@ -14,24 +15,104 @@ public static class WorkingWithText
         // Do not use .Sort, it will cause the test to pass when it actually does not.
         public static bool IsConsecutive(string hyphenNum)
         {
-            return default;
+            bool valid = false;
+
+            try
+            {
+                
+
+                List<int> list = hyphenNum.Split('-').Select(p => int.Parse(p)).ToList();
```

Comment:
When doing a 
```suggestion
                List<int> list = hyphenNum.Split('-').Select(p => int.Parse(p)).ToList();
```

You can instead fo

```suggestion
                List<int> list = hyphenNum.Split('-').Select(int.Parse).ToList();
```